shared/src/main/java/uk/gov/di/authentication/entity/TICFCRIRequest.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.di.authentication.entity;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+import uk.gov.di.authentication.shared.validation.Required;
+
+import java.util.List;
+
+public record TICFCRIRequest(
+        @Expose @Required @SerializedName("sub") String internalPairwiseId,
+        @Expose @Required @SerializedName("vtr") List<String> vtr,
+        @Expose @Required @SerializedName("govuk_signin_journey_id") String journeyId,
+        @Expose @Required @SerializedName("authenticated") String authenticated,
+        @Expose @SerializedName("initial_registration") String initialRegistration,
+        @Expose @SerializedName("password_reset") String passwordReset) {
```

Comment:
Both the @SerializedName annotations are redundant as the gson serializer has been configured to add underscores and lower case from variables named using camel case.